src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -60,6 +70,10 @@ public void fillFluidContainerItem(ActivateEvent event, EntityRef item, FluidCon
 
                         FluidUtils.setFluidForContainerItem(removedItem, fluidType);
 
+                        // Fill this fluid container up to max capacity.
```

Comment:
I originally placed this line here for the scenario in the future where the world (or object interacted with) doesn't have enough fluid to fill up the container to max capacity. But, I suppose I could just add an overloaded method for "setFluidForContainerItem" to account for that.